Dockerfile

```diff
@@ -1 +1,19 @@
+FROM python:3.8-alpine
+
+LABEL description="Alpine Linux with Python 3" \
+	  author="Oleksandr6676" \
+	  app="Core app" \
```

Comment:
Please, fix indentation. Then we can merge.

Reply:
Hmm, it looks fine in my code editor. Sure, I’ll fix it

Reply:
Interesting, you are right. So let it be then.